.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+<!--
+Thank you for contributing to TiDB Operator! Please read TiDB Operator's [CONTRIBUTING](https://github.com/pingcap/tidb-operator/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+-->
+
+### What problem does this PR solve? <!--add issue link with summary if exists-->
+
+### What is changed and how it works?
+
+### Check List <!--REMOVE the items that are not applicable-->
+
+Tests <!-- At least one of them must be included. -->
+
+ - Unit test
+ - E2E test
+ - Manual test (add detailed scripts or steps below)
+ - No code
+
+Code changes
+
+ - Has Helm charts change
+ - Has Go code change
+ - Has CI related scripts change
+
+Side effects
+
+ - Increased code complexity
```

Comment:
How is this information used?

Reply:
Hmm, this is copied from PD and TiDB repository. I think we share the same idea with these repos.

Reply:
I hear complaints about the existing templates. I think we can improve upon them by deleting things of dubious value and try to have a checklist of things that submitters must do or explain. Code complexity is very subjective and will need to be assessed independently by the reviewers ignoring of what is written here. If we really want this information in a useful objective form we could run a cyclomatic complexity linter.